test/bdd/pkg/cli/cli_steps.go

```diff
@@ -90,6 +92,27 @@ func (e *Steps) checkCreatedDID() error {
 	return nil
 }
 
+func (e *Steps) checkRecoveredDID() error {
+	const numberOfPublicKeys = 2
+
+	const numberOfServices = 2
+
+	doc, err := e.resolveDID(e.createdDID.ID)
+	if err != nil {
+		return err
+	}
+
+	if len(doc.PublicKey) != numberOfPublicKeys {
+		return fmt.Errorf("did doc public key is not equal to 2")
+	}
+
+	if len(doc.Service) != numberOfServices {
```

Comment:
Also, you created DID had two services and two keys so I am not sure that this test checks properly. Can you check for different key id say "recovered-id"?

Reply:
done